packages/pwa-kit-react-sdk/package.json

```diff
@@ -54,11 +54,12 @@
     "minimatch": "3.0.4",
     "mkdirp": "^1.0.4",
     "prop-types": "^15.6.0",
-    "react-ssr-prepass": "^1.5.0",
     "pwa-kit-runtime": "^2.3.0-dev",
+    "react-ssr-prepass": "^1.5.0",
```

Comment:
I think you need to re-base this pr to be off of develop.

Reply:
It's best to do it after we could verify that everything works as expected as since the ssr of react-query is not in develop yet, there is not way to verify the work in this PR